app/components/rdf-input-fields/mandataris-status-selector.js

```diff
@@ -0,0 +1,23 @@
+import { queryRecord } from 'frontend-lmb/utils/query-record';
+import ConceptSchemeSelectorComponent from './concept-selector';
+import { MANDATARIS_AANGEWEZEN_STATE } from 'frontend-lmb/utils/well-known-uris';
+
+export default class RdfInputFieldsMandatarisStatusSelectorComponent extends ConceptSchemeSelectorComponent {
+  async fetchOptions(searchData) {
+    const statuses = await super.fetchOptions(searchData);
+    const mandatarisUri = this.storeOptions.sourceNode.value;
```

Comment:
this won't work if they change both the mandate and the status. You should fetch the current mandate from the form data and check if it has the burgemeester classification